managed-databases/document-database/how-to/connect-database-private-network.mdx

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: How to connect a Document Database Instance to a Private Network
+  description: This page explains how to connect a Document Database Instance to a Private Network
+content:
+  h1: How to connect a Document Database Instance to a Private Network
+  paragraph: This page explains how to connect a Document Database Instance to a Private Network
+tags: managed-database postgresql ferretdb mongodb private-network database-instance
+dates:
+  validation: 2023-08-01
+  posted: 2023-08-01
+categories:
+  - managed-databases
+---
+
+[Private Networks](/network/vpc/concepts/#private-networks) allows you to enhance the security of your system's architecture by isolating it from the Internet.
+
+When using Private Networks, your application connects directly to your Database nodes without going through a Load Balancer.
+
+This improves performance by reducing the latency between your application and your database nodes. It also increases the security of your databases, as Instances in your Private Network can directly communicate with your Database Instance, without passing through the public Internet.
+
+You can create new Database Instances to attach to your Private Network, or attach existing ones.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [Document Database Instance](/managed-databases/document-database/quickstart)
+  - You have [generated an API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+</Message>
+
+## How to attach a Database Instance to a Private Network
+
+1. Click **Document Databases** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Click the **Private Networks** tabs. The Private Network configuration displays.
+4. Click **+ Attach Private Network**. The Private Network configuration wizard displays.
+    - You can choose whether to:
+        - Select and attach an existing Private Network and select the network from the drop-down list.
+        - Create and attach a new Private Network and enter the name of the network in the form.
+
+    <Message type="important">
+      Your private IP addresses are automatically allocated. They are managed via an IP Address Manager (IPAM). You can retrieve them on the Private Network tab of your Database Instance. To allocate IPs manually, you can use the Scaleway API and CLI, or Terraform.
+    </Message>
+5. Click **Attach Private Network** to conclude.
+
+<Message type="tip">
+  If you are using Scaleway Kubernetes Kapsule, you can use the [scaleway-k8s-vpc](https://github.com/Sh4d1/scaleway-k8s-vpc) github project to connect a Database Instance to your pods using a Private Network.<br /><br />
```

Comment:
Is there any reason to use this (outdated and not for production-ready) project instead of the native Private Network feature?